code/parachain/runtime/dali/src/xcmp.rs

```diff
@@ -253,7 +253,7 @@ pub fn xcm_asset_fee_estimator(instructions: u8, asset_id: CurrencyId) -> Balanc
 	assert!((instructions as u32) <= MaxInstructions::get());
 	let total_weight = UnitWeightCost::get() * instructions as u64;
 	Trader::weight_to_asset(total_weight, asset_id)
-		.expect("use only in simulator")
+		.expect("use only in simulator") // Isn't it better to have a default value than potentially panic?
```

Comment:
this is for tests, may be move it somewhere. so it can be used later in RPC (sure uwnrap to be removed)

Reply:
best is to have None in code. and wrapper for tests with expect (tests expect fee asked for only sufficient codes)